cli/cmd/configure_monitoring.go

```diff
@@ -107,9 +110,42 @@ keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAM
 				return fmt.Errorf("Sending configure-monitoring event was unsuccessful. %s", *err.Message)
 			}
 
-			// if eventContext is available, open WebSocket communication
-			if eventContext != nil && !SuppressWSCommunication {
-				return websockethelper.PrintWSContentEventContext(eventContext, endPoint)
+			maxFetchEventRetries := 10
+			fetchEventRetryTime := 3
+			// if eventContext is available, try to fetch the correlating configure-monitoring.finished event
+			gotEvent := false
+			if eventContext != nil {
+				for i := 0; i < maxFetchEventRetries; i = i + i {
+					events, errObj := eventHandler.GetEvents(&apiutils.EventFilter{
+						KeptnContext: *eventContext.KeptnContext,
+						EventType:    keptnv2.GetFinishedEventType("configure-monitoring"),
+					})
+					if errObj == nil && len(events) > 0 {
+						for _, event := range events {
+							eventData := &keptnv2.EventData{}
+							decoder, err := mapstructure.NewDecoder(&mapstructure.DecoderConfig{
+								Squash: true,
+								Result: eventData,
+							})
+							if err != nil {
+								return fmt.Errorf("could not decode event: " + err.Error())
+							}
+
+							if err := decoder.Decode(event.Data); err != nil {
+								return fmt.Errorf("could not decode event: " + err.Error())
+							}
+
+							logging.PrintLog(eventData.Message, logging.InfoLevel)
+						}
+						gotEvent = true
+						break
+					}
+					<-time.After(time.Duration(fetchEventRetryTime) * time.Second)
+				}
+
+				if !gotEvent {
+					logging.PrintLog("Could not retrieve configure-monitoring.finished event with KeptnContext "+*eventContext.KeptnContext+". Please check your monitoring solution if it has been configured correctly.", logging.InfoLevel)
+				}
```

Comment:
This whole new block is actually a function that just tries to retrieve an event based on some timeout and retry settings. I think it would be better suited in a helper function, maybe even in go-utils?
Any thoughts on this @agrimmer @bacherfl ?

Reply:
I have made a PR for the go-utils where I have added this functionality: https://github.com/keptn/go-utils/pull/221